packages/calling/src/CallingClient/constants.ts

```diff
@@ -42,7 +42,7 @@ export const DUMMY_METRICS = {
   },
 };
 export const DUMMY_MOBIUS_URL = 'https://mobius.aintgen-a-1.int.infra.webex.com/api/v1';
-export const FETCH_NAME = /^[a-zA-Z ]+/;
+export const FETCH_NAME = /^[a-zA-Z ()-]+/; // Match only alphabets, space, parenthesis, and hyphen. Ignores rest of the chars as soon a non matching char is found.
```

Comment:
could we rename this as regex?
```suggestion
export const FETCH_NAME_REGEX = /^[a-zA-Z ()-]+/; // Match only alphabets, space, parenthesis, and hyphen. Ignores rest of the chars as soon a non matching char is found.
```

Reply:
Removed usage of this regex